moe/library/lib_item.py

```diff
@@ -231,7 +243,11 @@ class PathType(sqlalchemy.types.TypeDecorator):
 
     library_path: Path  # will be set on config initialization
 
-    def process_bind_param(self, value, dialect):
+    def process_bind_param(
+        self,
+        value: Path | None,
+        dialect: sqlalchemy.engine.interfaces.Dialect,  # noqa: ARG002
+    ) -> str | None:
```

Comment:
I believe this syntax requires Python 3.10+ (will reference this in my main review)

Reply:
Importing `annotations` from `__future__` allows this in python 3.9.